RAD-Seq/README.md

```diff
@@ -0,0 +1,127 @@
+RAD-Seq data analysis
+======================
+
+RAD-Sequencing is a reduced representation library method used to obtain genomic information on a whole genome without sequencing each nucleotides. Here, we propose some materials to learn how to analyze RAD-Seq data.
+
+# Slides
+
+Several deck of slides are available for this topic:
+
+- [General introduction about RNA seq data analysis](http://bgruening.github.io/training-material/RAD-Seq/slides/)
+- Slide deck related to the tutorials:
+    - [Reference-based RAD-Seq data analysis](http://bgruening.github.io/training-material/RAD-Seq/slides/ref_based.html)
+
+# Tutorials
+
+A tutorial with hands-on is available for this topic:
+
+- [Reference-based RAD-Seq data analysis](tutorials/ref_based.md)
```

Comment:
I was planning to make a first almost "empty" RAD-seq section then fill it with tuto, docker, and others amterials.. but maybe it's better to create / fill completely the rad-seq training material part before PR ? In the meantime, I have to modify this line ;) thanks again !!!!